daemon/algod/api/generate.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -e
+
+rootdir=`dirname $0`
+pushd $rootdir
+
+# Convert v2 to v3
+curl -s -X POST "https://converter.swagger.io/api/convert" -H "accept: application/json" -H "Content-Type: application/json" -d @./algod.oas2.json  -o 3.json
```

Comment:
this would fail without internet connection. it might be acceptable compromise, as it's needed only when we regenerate the apis.

Reply:
The expectation is that the REST endpoints will change infrequently, and only during development. So this wont be a significant burden.